apps/www/components/Blog/BlogListItem.tsx

```diff
@@ -30,7 +30,7 @@ const BlogListItem = ({ post }: Props) => {
           <div className="flex flex-col space-y-6">
             <div className="flex flex-col space-y-3">
               <div
-                className={`border-scale-300 relative mb-4 h-60 w-full overflow-auto rounded-lg border shadow-sm`}
+                className={`border-border relative mb-4 h-60 w-full overflow-auto rounded-lg border shadow-sm`}
```

Comment:
can just be `border`

are you thinking find and replace will be hard?

Reply:
But let's say you only want `border-t`, you would have to write `border-t border-border`. If you write `border-t border`, the second "border" would set the border on all 4 sides.

Reply:
So we could just probably have the shorthand `border` when it's all 4 borders and we know it's the default color.

Reply:
But it could be actually `border-default` instead of `border-border`. Much better...

Reply:
I think we should stay away from double barrel class names all together, `border-border` will give me nightmares

Reply:
> border-border is the same as border right?
Not exactly. `border` is [this tailwind utility](https://tailwindcss.com/docs/border-width#all-sides) to set all sides.

Reply:
I agree `border-border` is very ugly. But `border-default` should be a good solution.

Reply:
Hm for some reason `border-DEFAULT` doesn't work though...

Reply:
These don't produce any output:

<img width="1028" alt="Screenshot 2023-10-23 at 18 37 31" src="https://github.com/supabase/supabase/assets/25671831/4a9bfae4-1237-46e7-b946-b8b40ace4cfe">

This one does:

<img width="435" alt="Screenshot 2023-10-23 at 18 37 10" src="https://github.com/supabase/supabase/assets/25671831/37572c39-0852-4205-bc40-3c72d01fcd4f">